src/pages/ReimbursementAccount/PersonalInfo/PersonalInfo.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {forwardRef, useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useSubStep from '@hooks/useSubStep';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import getInitialSubstepForPersonalInfo from '@pages/ReimbursementAccount/utils/getInitialSubstepForPersonalInfo';
+import getSubstepValues from '@pages/ReimbursementAccount/utils/getSubstepValues';
+import * as BankAccounts from '@userActions/BankAccounts';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount, ReimbursementAccountDraft} from '@src/types/onyx';
+import Address from './substeps/Address';
+import Confirmation from './substeps/Confirmation';
+import DateOfBirth from './substeps/DateOfBirth';
+import FullName from './substeps/FullName';
+import SocialSecurityNumber from './substeps/SocialSecurityNumber';
+
+type PersonalInfoOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** The draft values of the bank account being setup */
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
+};
+
+type PersonalInfoProps = PersonalInfoOnyxProps & {
+    /** Goes to the previous step */
+    onBackButtonPress: () => void;
+
+    /** Exits flow and goes back to the workspace initial page */
+    onCloseButtonPress: () => void;
+};
+
+const bodyContent: Array<React.ComponentType<SubStepProps>> = [FullName, DateOfBirth, SocialSecurityNumber, Address, Confirmation];
```

Comment:
NAB: I just don't like this syntax, so I'd extract it like this:
```suggestion
type SubStepComponent = React.ComponentType<SubStepProps>;
const bodyContent: SubStepComponent[] = [FullName, DateOfBirth, SocialSecurityNumber, Address, Confirmation];
```

Reply:
this syntax is not allowed
`ESLint: Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.(@typescript-eslint/array-type)`

Reply:
If you extract it like I've suggested it will work. This eslint rule forbids using complex types (like generics) with T[] syntax, because this is hard to read: `React.ComponentType<SubStepProps>[]`. `SubStepComponent[]` will is allowed